src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -32,7 +32,6 @@ const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
   let template;
   // @debt remove backButton from Navbar
   let hasBackButton = true;
-  let fullscreen = false;
```

Comment:
What's the intent/justification for removing this? Have we tested that it looks correct on all of the other templates that didn't previously set this to `true`? eg.:

- `VenueTemplate.jazzbar`
- `VenueTemplate.friendship`
- `VenueTemplate.partymap` / `VenueTemplate.themecamp`
- `VenueTemplate.artpiece`
- `VenueTemplate.conversationspace`
- `VenueTemplate.firebarrel`
- etc

Reply:
have checked it and reverted. Fixed for some cases

Reply:
Sounds good.